libc/src/string/memory_utils/x86_64/inline_memset.h

```diff
@@ -8,8 +8,8 @@
 #ifndef LLVM_LIBC_SRC_STRING_MEMORY_UTILS_X86_64_INLINE_MEMSET_H
 #define LLVM_LIBC_SRC_STRING_MEMORY_UTILS_X86_64_INLINE_MEMSET_H
 
-#include "src/__support/macros/attributes.h" // LIBC_INLINE
-#include "src/__support/macros/config.h"
+#include "src/__support/macros/attributes.h" // LIBC_INLINE_VAR
```

Comment:
`LIBC_INLINE` is also used from this header.

```suggestion
#include "src/__support/macros/attributes.h" // LIBC_INLINE, LIBC_INLINE_VAR
```